playbooks/validate.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Perform global variables sanity check
```

Comment:
Where possible I think it would be nice to avoid terms with too much mental health stigma. Can we say "Perform global variables validation"? I think that's a better fit since this is `validate.yml`. Thoughts?